wordpress-mobile/WordPress-Android
# Issue/8641 update permission dialog

### Fix
Update the permission dialog shown when a new site is created as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8641.  See the screenshots below for illustration.

![v2_sites_permission](https://user-images.githubusercontent.com/3827611/49157688-054afb00-f2e6-11e8-95e6-21096c237faa.png)

### Test
1. Go to ***Sites*** tab.
2. Tap ***Switch Site*** button.
3. Tap ***+*** action.
4. Tap ***Create WordPress.com site*** option.
5. Follow site creation steps.
6. Tap ***Configure*** button.
7. Notice ***Quick Start*** dialog is shown.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.
